File: online-judge-solutions/Codeforces/440A.cpp
//https://codeforces.com/problemset/problem/440/A
//Lang		:	GNU C++14
//Time		:	46 ms
//Memory	:	400 KB
#include<iostream>
#include<vector>
#include<algorithm>
using namespace std;
#define fastInp cin.tie(0); cout.tie(0); ios_base::sync_with_stdio(0);
int main(){
	fastInp;
	int n;cin>>n;
	vector<int> a(n,1000000);
	for(int i=0;i<n;i++) cin>>a[i];
	sort(begin(a),end(a));
	for(int i=0;i<n;i++) if(a[i]!=i+1) {cout<<i+1<<endl;break;}
	return 0;
}

File: online-judge-solutions/Codeforces/440A.py
#https://codeforces.com/problemset/problem/440/A
#Lang		:	Python 3.8
#Time		:	124 ms
#Memory         :	6800 KB
n=int(input())
a=list(map(int,input().split()))
a.append(1000000)
a.sort()
print([i+1 for i,x in enumerate(a) if(x!=i+1)][0])
